src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -366,6 +366,10 @@
                                      :binding "t f"
                                      :fn      ui-handler/toggle-contents!}
 
+   :command/add-to-favorites       {:desc    "Add to/remove from favorites"
+                                    :binding "mod+shift+f"
+                                    :fn      #((page-handler/toggle-favorite!))}
```

Comment:
@devonzuegel `page-handler/toggle-favorite!` is enough here, no need to wrap it :)
I've made changes in this commit https://github.com/logseq/logseq/commit/4270dbc755cab2c07b73bd7f64cd0da64d759457.

Reply:
oops good catch! thanks